packages/drivers/odsp-driver/src/zipItDataRepresentationUtils.ts

```diff
@@ -0,0 +1,416 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
Can you please add some reference to ZipIt documentation here?

Reply:
Done